mergify_engine/clients/github.py

```diff
@@ -431,6 +431,7 @@ async def items(
         oauth_token: typing.Optional[github_types.GitHubOAuthToken] = None,
         list_items: typing.Optional[str] = None,
         params: typing.Optional[typing.Dict[str, str]] = None,
+        page_limit: int = 100,
```

Comment:
Wondering if we should decrease that in another PR.
I'm having a hard time imagining what will work with 100 pages, whatever the request is.

Reply:
Agreed!

Reply:
I will propose another PR to fix that.